Cubical/Algebra/CommRing/Integers.agda

```diff
@@ -2,22 +2,298 @@
 module Cubical.Algebra.CommRing.Integers where
 
 open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.SIP
+open import Cubical.Foundations.Equiv
+open import Cubical.Foundations.Isomorphism
 
-open import Cubical.Algebra.CommRing
-open import Cubical.HITs.Ints.BiInvInt
-  renaming (
-    _+_ to _+ℤ_;
-    -_ to _-ℤ_;
-    +-assoc to +ℤ-assoc;
-    +-comm to +ℤ-comm
+open import Cubical.Reflection.Base using (_$_) -- TODO: add this to Foundation.Function
```

Comment:
would that be possible to add to `Foundation.Function`?

Reply:
Yes, that would be good!